app/javascript/app/components/header/header-styles.scss

```diff
@@ -1,29 +1,8 @@
-.header {
-  margin: 10px 0 0;
-  padding: 10px 0;
-  border-bottom: 1px solid #eee;
-}
-
-.link {
-  display: inline-block;
-  margin: 0 10px 0 0;
-  text-decoration: none;
+@import '../../styles/settings';
 
-  &Active {
-    text-decoration: underline;
-  }
-
-  &:hover {
-    color: #ccc;
-  }
-}
-
-.nav {
-  padding: 0;
-  list-style: none;
-  display: flex;
-}
-
-.navItem {
-  margin-right: 10px;
+.header {
+  background-color: $theme-color;
+  min-height: 300px;
```

Comment:
I thought the designs might change so it would be easier to change the css than the component.